[BUG] npm version command issuing invalid git command to check repository status

### Current Behavior:
I am running the npm version command but it is issuing an invalid git command to check whether the repository is clean and therefore always fails.

### Expected Behavior:
npm should either check the git version or issue an alternate --porcelain syntax to work properly with git, or report which version of git it requires to operate correctly.

### Steps To Reproduce:
1. In the environment shown below
2. Run 'npm version patch -m "release Version patch %s, now with double cleaning action." '
3. See error:
npm ERR! code 129
npm ERR! command failed
npm ERR! command /usr/bin/git status --porcelain=v1 -uno
npm ERR! error: option `porcelain' takes no value
npm ERR! usage: git status [options] [--] <pathspec>...
npm ERR! 
npm ERR!     -v, --verbose         be verbose
npm ERR!     -s, --short           show status concisely
npm ERR!     -b, --branch          show branch information
npm ERR!     --porcelain           machine-readable output

### Environment:
- OS: Ubuntu 14.04
- Node: 12.19.0
- npm: 7.5.6
- git: 1.9.1

Note, when I run the command /usr/bin/git status --porcelain -uno
instead, it correctly shows the status of the repo so it seems npm is expecting some version of git without verifying that it has that version.

--- a/docs/lib/content/commands/npm-version.md
+++ b/docs/lib/content/commands/npm-version.md
@@ -25,6 +25,8 @@ If run in a git repo, it will also create a version commit and tag.
 This behavior is controlled by `git-tag-version` (see below), and can be disabled on the command line by running `npm --no-git-tag-version version`.
 It will fail if the working directory is not clean, unless the `-f` or `--force` flag is set.
 
+**Note:** Git integration requires a reasonably recent version of git (2.0.0 or later is recommended). If you encounter issues with git commands, ensure your git installation is up to date.
+
 If supplied with `-m` or [`--message` config](/using-npm/config#message) option, npm will use it as a commit message when creating a version commit.
 If the `message` config contains `%s` then that will be replaced with the resulting version number.
 For example:

